lib/strings.c

```diff
@@ -35,9 +35,11 @@ static const char* const errorStrings[] = {
     "Out of memory.",                                 /* KTX_OUT_OF_MEMORY */
     "Transcoding of block compressed texture failed.",/* KTX_TRANSCODE_FAILED */
     "Not a KTX file.",                                /* KTX_UNKNOWN_FILE_FORMAT */
-    "Texture type not supported.",      /* KTX_UNSUPPORTED_TEXTURE_TYPE */
+    "Texture type not supported.",                    /* KTX_UNSUPPORTED_TEXTURE_TYPE */
```

Comment:
Where is KTX_UNSUPPORTED_TEXTURE_TYPE still used?

Reply:
It has around a dozen uses like:
On ktxFormatSize_initFromDfd failure
In _ktxUnpackETC if the format is not supported
In ktxTexture_GLUploadPrivate